resource/tasks.json

```
@@ -3647,6 +3647,10 @@
             [
                 "特米米.+",
                 "特米米"
+            ],
+            [
+                "崖心8",
```

```suggestion
                "崖心.+",
```
This is more consistent, in case other chars appear.